### ISSUE TITLE
Panics with "unreachable resource trampoline"

### ISSUE THREAD

**Issue by calvinrp** [bytecodealliance/jco#517]
I have a few components that run properly with wasmtime but panic with jco.

Easiest way to reproduce, build https://github.com/sunfishcode/hello-wasi-http

then run
```bash
jco serve target/wasm32-wasip1/release/hello_wasi_http.wasm
```

Panics here: https://github.com/bytecodealliance/jco/blob/main/crates/js-component-bindgen/src/transpile_bindgen.rs#L879

**Comment by guybedford** [comment]
This sounds like a bug, thanks for sharing, will look into it when I can.

**Comment by guybedford** [comment]
Managed to track this one down, fix released in 1.7.2. Thanks for the clear replication.
